autosklearn/automl.py

```diff
@@ -421,7 +414,8 @@ def fit(
         self._logger.debug('  resampling_strategy: %s', str(self._resampling_strategy))
         self._logger.debug('  resampling_strategy_arguments: %s',
                            str(self._resampling_strategy_arguments))
-        self._logger.debug('  shared_mode: %s', str(self._shared_mode))
+        self._logger.debug('  n_jobs: %s', str(self._n_jobs))
+        self._logger.debug('  dask_client: %s', str(self._dask_client))
```

Comment:
Do You think it also make sense to print the scheduler info? Here is the comparison:
```
>>> c = dask.distributed.Client()
>>> print(c)
<Client: 'tcp://127.0.0.1:40415' processes=4 threads=12, memory=16.69 GB>
```
vs
```
>>> print(c.scheduler_info())
{'type': 'Scheduler', 'id': 'Scheduler-6120002b-f080-4c31-a217-67cfea0be3b8', 'address': 'tcp://127.0.0.1:40415', 'services': {'dashboard': 8787}, 'workers': {'tcp://127.0.0.1:35037': {'type': 'Worker', 'id': 2, 'host': '127.0.0.1', 'resources': {}, 'local_directory': '/home/chico/work/auto-sklearn_prdask/dask-worker-space/dask-worker-space/worker-ret8pbbf', 'name': 2, 'nthreads': 3, 'memory_limit': 4171328512, 'last_seen': 1600200774.5238085, 'services': {'dashboard': 43551}, 'metrics': {'cpu': 6.0, 'memory': 52838400, 'time': 1600200774.516168, 'read_bytes': 4357.279003306836, 'write_bytes': 3565.2278291709317, 'num_fds': 24, 'executing': 0, 'in_memory': 0, 'ready': 0, 'in_flight': 0, 'bandwidth': {'total': 100000000, 'workers': {}, 'types': {}}}, 'nanny': 'tcp://127.0.0.1:45709'}, 'tcp://127.0.0.1:40489': {'type': 'Worker', 'id': 3, 'host': '127.0.0.1', 'resources': {}, 'local_directory': '/home/chico/work/auto-sklearn_prdask/dask-worker-space/dask-worker-space/worker-tn8ppdc0', 'name': 3, 'nthreads': 3, 'memory_limit': 4171328512, 'last_seen': 1600200774.523963, 'services': {'dashboard': 34521}, 'metrics': {'cpu': 6.0, 'memory': 52699136, 'time': 1600200774.5164387, 'read_bytes': 4009.613552863556, 'write_bytes': 3214.8973369440196, 'num_fds': 24, 'executing': 0, 'in_memory': 0, 'ready': 0, 'in_flight': 0, 'bandwidth': {'total': 100000000, 'workers': {}, 'types': {}}}, 'nanny': 'tcp://127.0.0.1:32925'}, 'tcp://127.0.0.1:41667': {'type': 'Worker', 'id': 0, 'host': '127.0.0.1', 'resources': {}, 'local_directory': '/home/chico/work/auto-sklearn_prdask/dask-worker-space/dask-worker-space/worker-i6y9l53r', 'name': 0, 'nthreads': 3, 'memory_limit': 4171328512, 'last_seen': 1600200774.5185788, 'services': {'dashboard': 35937}, 'metrics': {'cpu': 4.0, 'memory': 52658176, 'time': 1600200774.515757, 'read_bytes': 4008.394923610346, 'write_bytes': 3213.9202432941665, 'num_fds': 24, 'executing': 0, 'in_memory': 0, 'ready': 0, 'in_flight': 0, 'bandwidth': {'total': 100000000, 'workers': {}, 'types': {}}}, 'nanny': 'tcp://127.0.0.1:44841'}, 'tcp://127.0.0.1:42781': {'type': 'Worker', 'id': 1, 'host': '127.0.0.1', 'resources': {}, 'local_directory': '/home/chico/work/auto-sklearn_prdask/dask-worker-space/dask-worker-space/worker-uzr_gyvv', 'name': 1, 'nthreads': 3, 'memory_limit': 4171328512, 'last_seen': 1600200774.5124593, 'services': {'dashboard': 43121}, 'metrics': {'cpu': 4.0, 'memory': 52826112, 'time': 1600200774.0123239, 'read_bytes': 12385.847500842423, 'write_bytes': 12117.59090392746, 'num_fds': 24, 'executing': 0, 'in_memory': 0, 'ready': 0, 'in_flight': 0, 'bandwidth': {'total': 100000000, 'workers': {}, 'types': {}}}, 'nanny': 'tcp://127.0.0.1:46601'}}}

```

Reply:
I'm not sure. While running, the scheduler information will be available as a scheduler file in machine readable format anyway (https://github.com/automl/SMAC3/blob/development/smac/tae/dask_runner.py#L115), so I'm not 100% sure if there's additional helpful information here. If there's any, please let me know.